xml/containers-build.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha-containers-build"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Building Containers</title><date>2020-07-27</date>
+ </info>
+ <para>
+  This chapter documents various technologies and tools which can be used to
+  create containers. Most of the tools mentioned below are part of the
+  <emphasis role="strong">&sls; 15 SP2 Containers Module</emphasis>. You can
+  see the full list of packages in the <emphasis role="strong">Containers
+  Module</emphasis> in the &suse;
+  <link xlink:href="https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&amp;version=15.2&amp;arch=x86_64&amp;query=&amp;module=1963">Customer
+  Center</link>.
+ </para>
+ <sect1 xml:id="id-build-tools-available-to-customers">
+  <title>Build Tools Available to Customers</title>
+
+  <sect2 xml:id="id-docker">
+   <title>Docker</title>
+   <para>
+    Docker is many part system for creating and managing containers, it’s
+    main part being the <emphasis role="strong">Docker Open Source
+    Engine</emphasis> - a lightweight virtualization solution to run containers
+    simultaneously on a single control host. Docker containers are defined in
+    Dockerfiles, which describe how to assemble a private filesystem for a
+    container, and can also contain some metadata describing how to run the
+    container. Most Dockerfiles start from a parent image. For an introduction
+    on how to create custom base &sle; docker images see
+    <link xlink:href="https://documentation.suse.com/sles/15-SP1/single-html/SLES-dockerquick/">the
+    Docker Open Source Engine Guide</link>.
```

Comment:
After integrating with DC-SLES-all, please check for this type of link and use xrefs instead. E.g., use `<xref linkend="book-sles-docker"/>` here.